### PR TITLE
Add Number.parseFloat and Number.parseInt

### PR DESCRIPTION
**Pull request by existentialism**
Fixes https://github.com/kangax/compat-table/issues/1148

### CODE REVIEW

File: data-es6.js [hunk 1]
```diff
@@ -16210,6 +16210,40 @@ exports.tests = [
...
+        node0_12: true,
+        safari9: true,
+        typescript: typescript.corejs,
+      },
+    },
+    {
+      name: 'Number.parseInt',
+      mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseInt',
+      exec: function () {/*
+        return typeof Number.parseInt === 'function';
```
Comment (ljharb) [inline data-es6.js]: this should also test that `Number.parseInt === global.parseInt`, which the spec [requires](https://tc39.github.io/ecma262/#sec-number.parseint)

File: data-es6.js [hunk 2]
```diff
@@ -16210,6 +16210,40 @@ exports.tests = [
       },
     },
     {
+      name: 'Number.parseFloat',
+      mdn: 'https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/parseFloat',
+      exec: function () {/*
+        return typeof Number.parseFloat === 'function';
```
Comment (ljharb) [inline data-es6.js]: this should also test that `Number.parseFloat === global.parseFloat`, which the spec [requires](https://tc39.github.io/ecma262/#sec-number.parsefloat)

**Review by ljharb — CHANGES_REQUESTED** [review]
LGTM, just two comments :-)

**Comment by existentialism** [comment]
@ljharb thanks, updated!

**Review by ljharb — APPROVED** [review]
Please reconfirm that every engine in this list passes the new tests :-)

**Review by chicoxyzzy — CHANGES_REQUESTED** [review]
`Number.parseFloat === global.parseFloat` and `Number.parseInt === global.parseInt` parts are failing in all browsers. I think we should use `Function('return this')()` or something like that to get global object.

**Comment by ljharb** [comment]
oh, well yeah `global` isn't available by default - I agree with your suggestion (altho i think we could also create a global object variable using `Function('return this')()` in all the skeletons, and just rely on that existing in tests)

**Comment by zloirock** [comment]
I think that something like `Number.parseFloat === parseFloat` is enough.

**Comment by ljharb** [comment]
The extra checks can't hurt; and there's a proposal to change some behavior of these methods, so it'll be important to cover the details.

**Pull request merged**